test/testcmdlineparser.cpp

```diff
@@ -1713,65 +1708,13 @@ class TestCmdlineParser : public TestFixture {
         REDIRECT;
         const char * const argv[] = {"cppcheck", "--errorlist"};
         ASSERT(parser->parseFromArgs(2, argv));
-        ASSERT(parser->getShowErrorMessages());
-        ASSERT_EQUALS("", logger->str()); // empty since it is logged via ErrorLogger
-        ASSERT_EQUALS("", GET_REDIRECT_OUTPUT); // empty since the logging happens later on
-    }
-
-    void errorlist2() {
-        REDIRECT;
-        const char * const argv[] = {"cppcheck", "--errorlist"};
-        ASSERT(parser->fillSettingsFromArgs(2, argv));
-        ASSERT(parser->getShowErrorMessages());
         ASSERT_EQUALS("", logger->str()); // empty since it is logged via ErrorLogger
         ASSERT(startsWith(GET_REDIRECT_OUTPUT, "<?xml"));
         ASSERT(endsWith(GET_REDIRECT_OUTPUT, "</results>\n"));
     }
 
     // TODO: test --errorlist with product name
 
-    void errorlistverbose1() {
```

Comment:
No longer necessary since we no longer rely on `--verbose` which actually never had any effect at all. Possibly a very old left-over. I did not check the history when this changed.